app/src/main/java/com/github/multimatum_team/multimatum/DeadlineDetailsActivity.kt

```diff
@@ -56,8 +49,18 @@ class DeadlineDetailsActivity : AppCompatActivity() {
                 detailView.text = getString(R.string.isAlreadyDue)
             }
             else -> {
-                detailView.text =
-                    getString(R.string.DueInXDays, actualDate.until(date, ChronoUnit.DAYS).toString())
+                val remainingTime = actualDate.until(date, ChronoUnit.DAYS)
+                detailView.text = if (remainingTime <= 0) {
+                    getString(
+                        R.string.DueInXHours,
+                        actualDate.until(date, ChronoUnit.HOURS).toString()
```

Comment:
Codeclimate fails because of code duplication. To solve that, it will be good to store R.string.DueInXHours and
actualDate.until.... in val to avoid this warning.

Reply:
Unfortunatly, once I recuperate DueInXHours and the other time DueInXDays, it is the same for actualDate where once I recuperate the hours left and the other the days left. I can perfectly separate everything in two value that I set differently on an If but it will make the code longer and, in my opinion, will not make the code more understandable.
But if you think it is necessary, I will do it.

Reply:
I have put them in val and, infortunatly, this is still an issue for codeclimate

Reply:
Seeing that this issue is really minor but codeclimate seems to put to much importance on it, I have increased its 
threshold on similar code